src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+* FEATURE: `ConstructMultilineContextSnippet` will retrieve a few character after/before to prevent entire file when the file is one line only. [#2288](https://github.com/microsoft/sarif-sdk/pull/2288)
```

Comment:
>* FEATURE: `ConstructMultilineContextSnippet` [](start = 0, length = 45)

Is this part of this particular pull request?

Reply:
no...i was just adding missing release history stuff

---
In reply to: [581466908](https://github.com/microsoft/sarif-sdk/pull/2292#discussion_r581466908) [](ancestors = 581466908)